pynamodb/attributes.py

```diff
@@ -525,6 +525,25 @@ def deserialize(self, value):
         return json.loads(value)
 
 
+class IntegerAttribute(Attribute):
+    """
+    An integer attribute; unlike, NumberAttribute, truncates to an integer
+    """
+    attr_type = NUMBER
+
+    def serialize(self, value):
+        """
+        Encode integers as JSON
+        """
+        return json.dumps(int(value))
+
+    def deserialize(self, value):
+        """
+        Decode numbers from JSON
+        """
+        return int(json.loads(value))
+
```

Comment:
You may need to add a `get_value` method too

Example from Boolean attribute
```
     def get_value(self, value):
         # we need this for legacy compatibility.
         # previously, BOOL was serialized as N
         value_to_deserialize = super(BooleanAttribute, self).get_value(value)
         if value_to_deserialize is None:
             value_to_deserialize = json.loads(value.get(NUMBER_SHORT, '0'))
         return value_to_deserialize
```

`get_value` is called in various places  like `update` and the Integer behaviour should be consistent in all places

Reply:
Are you sure? In `Boolean` it's used as a migration path from `LegacyBoolean` which used a different underlying DynamoDB type. In my case, if someone replaces `NumberAttribute` with `IntegerAttribute`, they'll be using the same underlying type.

Reply:
I doubled checked, it seems all the places that uses `get_value`, pass it through `deserialize`. 
Example: https://github.com/pynamodb/PynamoDB/blob/57f5be33f9690bcf0c94b06efd6a7abd112edd03/pynamodb/models.py#L392
So I think you're right, it is not needed